receiver/mongodbatlasreceiver/internal/mongodb_atlas_client.go

```diff
@@ -553,3 +560,37 @@ func (s *MongoDBAtlasClient) processDiskMeasurementsPage(
 	}
 	return measurements.Measurements, hasNext(measurements.Links), nil
 }
+
+// retrieves the logs from the mongo API using API call: https://www.mongodb.com/docs/atlas/reference/api/logs/#syntax
+func (s *MongoDBAtlasClient) GetLogs(ctx context.Context, groupID, hostname, logName string) (*bytes.Buffer, error) {
+	buf := bytes.NewBuffer([]byte{})
+	var start string
+	if s.endTime != "" {
```

Comment:
when does endTime ever get updated?
also, is it ok if start is empty when we pass it into the DateRangeOptions?

Reply:
Start is ok if we were to pass it in as empty, although on default it goes to 24 hours prior to the current timestamp which I don't think would be terrible, but it might be quite a few logs built up. However, I realize now that I am simply updating start instead of s.startTime, likewise with s.endTime, will be making changes to those momentarily